src/renderer/components/+workloads-pods/load-pods-from-all-namespaces.injectable.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import { getInjectable } from "@ogre-tools/injectable";
+import namespaceStoreInjectable from "../+namespaces/store.injectable";
+import showErrorNotificationInjectable from "../notifications/show-error-notification.injectable";
+import podStoreInjectable from "./store.injectable";
+
+const loadPodsFromAllNamespacesInjectable = getInjectable({
```

Comment:
Why this needs to be injectable? How previously it was worked?

Reply:
Previously, this function would live inside React component or in separate file with `namespaceStore` and `podStore` imported at the top of the file.